frontend/pages/index.js

```diff
@@ -111,121 +107,125 @@ export default function Home() {
+    await write?.();
+    // TODO: Set UI to successful minting page
   };
 
   return (
-    
     <ThemeProvider theme={darkTheme}>
-    <Particles
-      id="tsparticles"
-      init={particlesInit}
-      loaded={particlesLoaded}
-      options={{
-        background: {
-          color: {
-            value: "#000000",
-          },
-        },
-        fpsLimit: 120,
-        interactivity: {
-          detect_on: 'window',
-          events: {
-            onClick: {
-              enable: true,
-              mode: "push",
-            },
-            onHover: {
-              enable: true,
-              mode: "repulse",
-            },
-            resize: true,
-          },
-          modes: {
-            push: {
-              quantity: 4,
-            },
-            repulse: {
-              distance: 200,
-              duration: 0.4,
+      <div
+        style={{
+          position: "relative",
+          zIndex: "-1",
+        }}
+      >
+        <Particles
+          id="tsparticles"
+          init={particlesInit}
+          loaded={particlesLoaded}
+          options={{
+            background: {
+              color: {
+                value: "#000",
+              },
             },
-          },
-        },
-        particles: {
-          color: {
-            value: "#ffffff",
-          },
-          links: {
-            color: "#ffffff",
-            distance: 150,
-            enable: true,
-            opacity: 0.5,
-            width: 1,
-          },
-          collisions: {
-            enable: true,
-          },
-          move: {
-            directions: "none",
-            enable: true,
-            outModes: {
-              default: "bounce",
+            fpsLimit: 120,
+            interactivity: {
+              detect_on: "window",
+              events: {
+                // onClick: {
+                //   enable: true,
+                //   mode: "push",
+                // },
+                // onHover: {
+                //   enable: true,
+                //   mode: "repulse",
+                // },
+                resize: true,
+              },
+              modes: {
+                push: {
+                  quantity: 4,
+                },
+                repulse: {
+                  distance: 200,
+                  duration: 0.4,
+                },
+              },
             },
-            random: false,
-            speed: 4,
-            straight: false,
-          },
-          number: {
-            density: {
-              enable: true,
-              area: 800,
+            particles: {
+              color: {
+                value: "#fff4",
+              },
+              links: {
+                color: "#6a1b9a40",
+                distance: 150,
+                enable: true,
+                opacity: 0.5,
+                width: 1,
+              },
+              // collisions: {
+              //   enable: true,
+              // },
```

Comment:
This caused particles to start flying around super fast for some reason. Feel free to add back.